crates/model/src/lib.rs

```diff
@@ -240,8 +239,7 @@ impl Div<Price> for Contracts {
     fn div(self, rhs: Price) -> Self::Output {
         let mut btc = self.0 / rhs.0;
         btc.rescale(8);
-        Amount::from_str_in(&btc.to_string(), Denomination::Bitcoin)
-            .expect("Error computing BTC amount")
+        Amount::from_btc(btc.to_f64().expect("to fit in f64")).expect("Parse the BTC amount")
```

Comment:
This does the same thing underthehood.

Reply:
not quite - we avoid using `Decimal`s `Display` implementation, which is AFAICT not as straightforward as f64 (regular float).

Reply:
but good point that `Amount::from_btc` creates a `String`. I have overlooked that.